lld/test/wasm/no-strip-segment.ll

```diff
@@ -0,0 +1,61 @@
+; RUN: split-file %s %t
+; RUN: llc -filetype=obj --mtriple=wasm32-unknown-unknown -o %t/main.o %t/main.ll
+; RUN: llc -filetype=obj --mtriple=wasm32-unknown-unknown -o %t/liba_x.o %t/liba_x.ll
+; RUN: llc -filetype=obj --mtriple=wasm32-unknown-unknown -o %t/liba_y.o %t/liba_y.ll
+; RUN: llvm-ar rcs %t/liba.a %t/liba_x.o %t/liba_y.o
```

Comment:
Other tests that use `ar` archives do `rm -f foo.a` first.. should we do that here too?